components/admin_console/billing/billing_subscriptions.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+
+import './billing_subscriptions.scss';
+
+type Props = {
+
+};
+
+const BillingSubscriptions: React.FC<Props> = () => {
+    return (
+        <div className='wrapper--fixed BillingSubscriptions'>
+            <FormattedAdminHeader
+                id='admin.billing.subscription.title'
+                defaultMessage='Subscriptions'
+            />
+            <div className='admin-console__wrapper'>
+                <div className='admin-console__content'>
+                    <div style={{border: '1px solid #000', width: '100%', height: '81px', marginBottom: '20px'}}>
```

Comment:
Is styling done inline due to theming? if not, then I would recommend moving this to the scss file.